pymeasure/instruments/instrument.py

```diff
@@ -60,11 +60,11 @@ class DynamicProperty(property):
                    parameters.
     """
 
-    def __init__(self, fget=None, fset=None, fdel=None, doc=None, fget_params_list=[],
-                 fset_params_list=[], prefix="" ):
+    def __init__(self, fget=None, fset=None, fdel=None, doc=None, fget_params_list=None,
+                 fset_params_list=None, prefix="" ):
         super().__init__(fget, fset, fdel, doc)
-        self.fget_params_list = fget_params_list
-        self.fset_params_list = fset_params_list
+        self.fget_params_list = () if fget_params_list is None else fget_params_list
+        self.fset_params_list = () if fset_params_list is None else fset_params_list
```

Comment:
Minor nitpick: having an fset_params_*list* which is a *tuple* is a bit weird :sweat_smile: (it could also lead to people trying to `.append()` on it in the future. Either a list or using `fset_params` or (less favoured) `fset_params_sequence` would probably be better, maybe?

Reply:
Here, I used the term 'list' in the parameter name as a term in English and not as indication of python type, that is "a number of connected items or names written or printed consecutively, typically one below the other." (source: Google translator)
In python a list can be implemented to a `tuple` or `list`.
Internally, I used a tuple because the list is not intended to be modified and it is more memory efficient.
If user wants to pass its own parameter, he could use either a `list` or `tuple` in python meaning.

For my taste, I think it is enough clear and safe.

Reply:
> In python a list can be implemented either with a tuple or list.

Clarification: In python a **sequence** can be implemented either with a tuple or list **(or range)**. [docs](https://docs.python.org/3/library/stdtypes.html#sequence-types-list-tuple-range)

But yeah, it's probably clear enough and does not need to block this PR.